assets/js/components/Dashboard/PaMessageForm/SelectStationsAndZones/SelectZonesPage.tsx

```diff
@@ -211,7 +216,7 @@ const SelectZonesPage = ({
         <div className="filters-container">
           <div>Service type</div>
           <div className="filters">
-            {sortRoutes(Object.keys(initialRoutes)).map((routeID) => {
+            {routeFilterLabels.map((routeID) => {
```

Comment:
We're calling this "route labels" here but the individual items a "route ID" — the latter term seems to also be used in other places so maybe we should use it here instead of "labels".

Reply:
Good call. I also feel like `initialRoutes` didn't really mean much so I tried improving that too.